Arm/test/wrist_joint_test.cpp

```diff
@@ -63,22 +63,64 @@ TEST_CASE("Wrist joint system testing")
   }
 
   SECTION("3.1 should return zero when setting pitch to zero")
+  {
+    wrist_joint.SetPitchPosition(0);
+    CHECK(wrist_joint.GetPitchPosition() == 0);
+  }
+
+  SECTION("3.2 Should return 90 when setting pitch to 90")
+  {
+    wrist_joint.SetPitchPosition(90);
+    CHECK(wrist_joint.GetPitchPosition() == 90);
+  }
+
+  SECTION("3.3 Should return 180 when setting pitch to 180")
+  {
+    wrist_joint.SetPitchPosition(180);
+    CHECK(wrist_joint.GetPitchPosition() == 180);
+  }
+
+  SECTION("3.4 Should return 0 when setting pitch to -1")
+  {
+    wrist_joint.SetPitchPosition(-1);
+    CHECK(wrist_joint.GetPitchPosition() == 0);
+  }
+
+  SECTION("3.5 Should return 180 when setting pitch to 181")
+  {
+    wrist_joint.SetPitchPosition(181);
+    CHECK(wrist_joint.GetPitchPosition() == 180);
+  }
```

Comment:
Since we had unit tests written for going beyond minimum and maximum angles we were able to verify that the logic was working correctly while we were developing. This accidentally showed the power of test driven development where you write unit tests prior to writing out the logic. Congrats :)